analysis_options.yaml

```diff
@@ -1,2 +1,215 @@
+#    - always_specify_types        # this works agains general Dart style guides
+    - annotate_overrides
+    - avoid_annotating_with_dynamic
+#    - avoid_as                    # obsolete - this was considered good practice in Dart 1
+    - avoid_bool_literals_in_conditional_expressions
+    - avoid_catches_without_on_clauses
+    - avoid_catching_errors
+# Annoying rule when using classes as namespaces for constants which is discouraged by the Dart
+# style guide, but I prefer it over aliases in imports for disambiguation.
+#    - avoid_classes_with_only_static_members
+    - avoid_double_and_int_checks
+    - avoid_equals_and_hash_code_on_mutable_classes
+    - avoid_escaping_inner_quotes  # TODO(zoechi) not yet available
+    - avoid_field_initializers_in_const_classes
+    - avoid_function_literals_in_foreach_calls
+    - avoid_implementing_value_types
+    - avoid_init_to_null
+    - avoid_js_rounded_ints
+    - avoid_null_checks_in_equality_operators
+    - avoid_positional_boolean_parameters
+    - avoid_private_typedef_functions
+    - avoid_redundant_argument_values
+    - avoid_renaming_method_parameters
+    - avoid_return_types_on_setters
+    - avoid_returning_null
+    - avoid_returning_null_for_void
+    - avoid_returning_this
+    - avoid_setters_without_getters
+    - avoid_shadowing_type_parameters # personal preference
+    - avoid_single_cascade_in_expression_statements
+    - avoid_types_on_closure_parameters
+    - avoid_unnecessary_containers
+    - avoid_unused_constructor_parameters
+    - avoid_void_async
+    - await_only_futures
+    - camel_case_extensions
+    - camel_case_types
+    - cascade_invocations
+    - constant_identifier_names
+    - curly_braces_in_flow_control_structures # personal preference
+    - directives_ordering
+    - do_not_use_environment # depends on project requirements
+    - empty_catches
+    - empty_constructor_bodies
+    - exhaustive_cases
+    - file_names
+#    - flutter_style_todos         # I find the required ":" annoying and pointless
+    - implementation_imports
+    - join_return_with_assignment
+    - leading_newlines_in_multiline_strings
+    - library_names
+    - library_prefixes
+    - lines_longer_than_80_chars
+# Too annoying when long strings with interpolations are broken into multiple lines which we do often
+#    - missing_whitespace_between_adjacent_strings
+    - no_default_cases             # experimental
+    - no_runtimeType_toString
+    - non_constant_identifier_names
+    - null_closures
+    - omit_local_variable_types
+# annoying when introducing a class to be used as interface that has only one member
+#    - one_member_abstracts
+    - only_throw_errors
+    - overridden_fields
+    - package_api_docs
+    - package_prefixed_library_names
+    - parameter_assignments        # personal preference
+    - prefer_adjacent_string_concatenation
+    - prefer_asserts_in_initializer_lists
+#    - prefer_asserts_with_message # personal preference
```

Comment:
We can enable this

Reply:
I find it a bit noisy but I'm fine mit whatever the others want.

Reply:
I agree, lets keep this disabled.